### DISCUSSION TITLE
Registration Execution

### DISCUSSION THREAD

**Discussion by StonedCookieGaming** [masterking32/WoWSimpleRegistration#97] [General]
I am writing a website and was curious as to how you handled the execution of the registration process? Seems my code always causes an error in the database.

**Comment by victorxtx** [comment]
1. use super global $_POST - which is an array - to obtain Username and Password, etc.
2. Make sure what branch of core base you're using, find the anthentication database name.
3. Build an SQL string, like "INSERT INTO <auth_db_name> VALUES('<username>', '<password>', ...)" - This is not easy.
Note that different core use different encryption method, you should use the right method to encrypt the username and password.
For AZ core, the db field is like `salt` and `verifier`, you make php to generate a salt which is a string, and mix up salt, username and password, encrypt them with an algorithm called SRP6 - which is a function - and the function returns you a string, which should be written into the `verifier` field.
For TC core, things are easier, just concat <username>, ':' and <password>, use lib function sha1 to generate a string, write it into `sha_pass_hash` filed, done.
